tests/common.go

```diff
@@ -745,7 +745,12 @@ func PrintCommandOutput(cmnd string) {
 }
 
 func PrintSvPoolStatus(node node.Node) {
-	runCmdGetOutput("pxctl sv pool show", node)
+	output, err := runCmdGetOutput("pxctl sv pool show", node)
+	if err != nil {
+		log.Warnf("error getting pool data on node [%s], cause: %v", node.Name, err)
+		return
```

Comment:
Dont you want to return an error?

Reply:
it is just for logging. do not want fail with this.